instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/RouteLineLayersTest.kt

```diff
@@ -453,8 +453,9 @@ class RouteLineLayersTest : BaseTest<BasicNavigationViewActivity>(
             alternativeRoutesMetadata = mapboxNavigation.getAlternativeMetadataFor(routesUpdate)
         )
 
-        // the alternative route turns out to fully overlaps with the original route,
-        // hence the 1.0 offset
+        // NN doesn't let an alternative route to overlap with primary anymore.
+        // An overlapped alternative is failed to process and getAlternativeMetadataFor returns null for it
+        // what's expected result from route line in this case?
```

Comment:
@LukasPaczos , @cafesilencio can you please help with that?

Reply:
@VysotskiVadim we'll need to refactor the test case.

What we're trying to test for is a situation where an alternative route has a duplicate point. I'd recommend leaving the `route_response_japan_2` in place (as it has the duplicate) and change `route_response_japan_1`. As far as I can tell, `route_response_japan_1` actually contains two routes, so picking the second one instead of the first one from this response should do the trick.

Reply:
@LukasPaczos , what is duplicate point?

Reply:
It was easier to generate new routes and insert duplicating point in the alternative. @LukasPaczos , @cafesilencio , can you please verify the test?

Reply:
@VysotskiVadim that might lead to some inconsistencies down the line. A new point in the route shifts all of the reference indices for things like intersections, incidents, closures, annotations, etc. which you have not adjusted. I'd prefer leaving the old route in, as it was a real route that was returned by our backend, to keep the regression test as close to the production conditions as possible.

Reply:
It's a fair point. It wasn't easy but I updated primary route using `exclude=point(140.880777 38.250446)` to make not similar to existing alternative